kubernetes-model-generator/pom.xml

```diff
@@ -53,6 +53,10 @@
     <module>kubernetes-model-settings</module>
     <module>kubernetes-model-storageclass</module>
     <module>openshift-model</module>
+    <module>openshift-operator-model</module>
+    <module>openshift-operatorhub-model</module>
+    <module>openshift-monitoring-model</module>
+    <module>openshift-console-model</module>
```

Comment:
nit: naming of modules/directories/etc. is not consistent with the other naming patterns

>>>> `openshift-model-*`